src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import Logo from "../icons/Logo.js";
+
+const Title = styled.header`
+  height: 80px;
+  width: 375px;
+  background-color: rgb(0, 0, 0);
+  margin: 0;
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-image: linear-gradient(to right, rgba(37, 206, 197, 1), black) 50 15%;
+`;
+
+const Container = styled.div`
+  align-items: center;
+  margin: 15px 15px;
+  flex-direction: column;
+`;
+
+const Headline = styled.h1`
+  font-size: 25px;
+  color: rgb(37, 206, 197);
+  margin-left: 40px;
+  padding-top: 8px;
+  text-align: center;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serifw;
+`;
+
+const Menu = styled.img``;
+
+function Header() {
+  return (
+    <Title>
+      <Container>
+        <Logo />
+      </Container>
+      <Headline>EventListener</Headline>
+      <Menu></Menu>
+    </Title>
```

Comment:
<Title> seems to function as a wrapper for the Header component, you could name it accordingly, and use <Title> where <Headline> is right now. Also, <Container> is a little too generic in my opinion. "LogoWrapper" or "LogoContainer" would be bit more descriptive.

Reply:
Thx for the comment, i will rename the Component.